core/head/head.view.js

```diff
@@ -12,13 +12,9 @@ export class HeadView extends View {
 
 		this.drop = new Command({
 			canExecute: e => {
-				if (e.source && e.source.key === tagName) {
```

Comment:
Don't we need to check the tag name? It's kinda of convension to check if source of drag/drop is valid